data/shared/citizen/scripting/v8/console.js

```diff
@@ -0,0 +1,328 @@
+// Global console
```

Comment:
Is this file based on any third-party code with its own licensing specification? If so, said specification should be provided.